dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPI.java

```diff
@@ -235,13 +235,10 @@ public abstract void copy(Folder folderToCopy, Host newParentHost, User user, bo
 	/**
 	 * Gets a list of  'working'  Contentlet under given folder
 	 *
-	 * @param parent
-	 * @param user
-	 * @param respectFrontEndPermissions
-	 * @return
-	 * @throws DotStateException
-	 * @throws DotDataException
+	 * @deprecated use {@link com.dotmarketing.portlets.contentlet.business.ContentletAPI#findContentletsByFolder(Folder,
+	 * User, boolean)} instead
 	 */
+	@Deprecated
 	public abstract List<Contentlet> getContent(Folder parent, User user, boolean respectFrontEndPermissions) throws DotStateException,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)